redis_bitset.go

```diff
@@ -45,3 +50,25 @@ func (r *RedisBitSet) Test(offsets []uint) (bool, error) {
 
 	return true, nil
 }
+
+func (r *RedisBitSet) Delete() error {
+	n := uint(0)
+	keys := make([]string, 0)
+	for n <= r.maxN {
+		key := fmt.Sprintf("%s:%d", r.keyPrefix, n)
+		keys = append(keys, key)
+		n = n + 1
+	}
+	_, err := r.conn.Do("DEL", strings.Join(keys, " "))
+	return err
+}
```

Comment:
This will only delete the largest key you've seen, which won't be the case if loading an existing bitset on init. You could use KEYS and DEL alongside an transaction (or best effort). UNLINK may also be a reasonable alternative to DEL.